packages/now-build-utils/package.json

```diff
@@ -34,10 +36,12 @@
     "fs-extra": "7.0.0",
     "glob": "7.1.3",
     "into-stream": "5.0.0",
+    "jest": "24.9.0",
```

Comment:
I think `jest` is only in the root `package.json` since nearly all the other packages use it.
Can we do the same for `ts-jest` and solve this problem for all packages in the monorepo?